inventory/group_vars/all.yml

```diff
@@ -8,11 +7,1913 @@ dns_replace_address:
+      0234408.cn
+      024chaoren.cn
+      0302woeuxcbrnxcvsdfghu.xyz
+      0309fuweopermncvbnazdf.xyz
+      0313413.com
+      03196688.com
+      0531mnk.net
+      0551zgz.com
+      0571hy.com
+      0571yy.com
+      0923csfsteibv12.xyz
+      0937js.com
+      0bc.top
+      0ca0c1f060.com
+      0fwg.cn
+      0qkc.cn
+      1.bangdan5.com
+      1.ideng.com
+      1115potmgabfvixqwi.top
+      116zhuce.com
+      118ex.cn
+      125331.top
+      12gf.gold
+      12jf3.cn
+      12rge.xyz
+      13wnjs.com
+      15cf13b2000029.com
+      16guu.com
+      17bqg.top
+      183pu.cn
+      18dusun.com
+      1999sg.com
+      1d1px.net
+      1fv3b.cn
+      1kzh.com
+      1uandun.com
+      2012ui.com
+      210189.com
+      216pu.cn
+      21xsk.com
+      228pu.cn
+      27f35.cn
+      2byuf.cn
+      2fnrc.cn
+      2fx32.cn
+      2giga.download
+      2hqd0b.cn
+      2l1f.cn
+      2mdnsys.com
+      2q5p.cn
+      2scjbg.com
+      2shoushi.com
+      3.chuanyi5.com
+      333dm.com
+      360zuowen.net
+      365bibi.com
+      365yigou.cn
+      36ia2.cn
+      3abd17850c2.com
+      3amall.cn
+      3de4e583083.com
+      3wnp9.ru
+      3y0h.cn
+      3y6c.cn
+      40ic.cn
+      44-trk-srv.com
+      455e036e18.com
+      45le.cn
+      45xa.cn
+      46cq.cn
+      47n209.com
+      497761.top
+      4ebde70ccf.com
+      4ggww.com
+      4s5n.cn
+      51.77.227.100
+      51.77.227.101
+      51.77.227.102
+      51.77.227.103
+      51.77.227.96
+      51.77.227.97
+      51.77.227.98
+      51.77.227.99
+      51.89.187.136
+      51.89.187.137
+      51.89.187.138
+      51.89.187.139
+      51.89.187.140
+      51.89.187.141
+      51.89.187.142
+      51.89.187.143
+      5151gj.com
+      517xx.com
+      51posj.cn
+      52lubo.cn
+      53txs.com
+      5535aibo.com
+      555b2b.com
+      5a8p12.cn
+      5dian.org
+      5eln.cn
+      5hytr.xyz
+      5mbd2.com
+      5pb266.com
+      5t4o.cn
+      609623.com
+      618e0fb461a7.com
+      626uc.com
+      64ri.cn
+      652748.com
+      6612151.cn
+      6612152.cn
+      664gp.com
+      66992.top
+      66994.top
+      66san.com
+      67db4c378c1b.com
+      688ct.com
+      6945k6.cn
+      69lm.com.cn
+      69yll.cn
+      6f0ede26e60.com
+      6l1967.cn
+      6yush.cn
+      700ok.net
+      70e.me
+      70f4b25c15.com
+      72zh.cn
+      733xs.top
+      74p030.cn
+      74vc.cn
+      766ba.net
+      7687d91e8ad8.com
+      7891655.cn
+      7c5162f3211.com
+      7ia9h.cn
+      7jiajiao.com
+      7m0hj.cn
+      7n3e8o.com
+      7xz3.com
+      8368661.com
+      8790d6c89d60407aa77d.com
+      88android.com
+      88infra-strat.com
+      8j24h.cn
+      8v61j.cn
+      913vt.com
+      91luopan.com
+      922x7c.cn
+      931b4fb4be61b.com
+      9377if.com
+      93vitui.com
+      95105012.com
+      95p2.com
+      991pao.com
+      99tuan.com.cn
+      9h6qq.cn
+      9pb0h.cn
+      :34.66.206.188
+      :35.224.227.218
```

Comment:
What are these? They look funky and will probably break BIND in later stages.

Reply:
Added by Support & Operations for the DoH server originally.